### PR TITLE
adding 1.22 bundles

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: generatebundlefile/hack/generate_bundle.sh [hunk 1]
```diff
@@ -35,6 +35,17 @@ mkdir -p ${BASE_DIRECTORY}/bin
 make oras-install
 
 ECR_PASSWORD=$(aws ecr-public get-login-password --region us-east-1)
-cd output/
+cd ${BASE_DIRECTORY}/generatebundlefile/output
 ${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1-21-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
-${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1-21-latest" bundle.yaml
\ No newline at end of file
+${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1-21-latest" bundle.yaml
```
Comment (ewollesen) [inline generatebundlefile/hack/generate_bundle.sh:40]: This kind of cut and paste isn't very sustainable long term.

File: generatebundlefile/hack/release.sh [hunk 2]
```diff
@@ -45,7 +45,7 @@ ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
     --input ${BASE_DIRECTORY}/generatebundlefile/data/input_121.yaml \
     --release-profile ${PROFILE}
 
-# Create the prod bundle
+# Create the prod bundle for 1.21
 ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
```
Comment (vincentni) [inline generatebundlefile/hack/release.sh:49]: `{BASE_DIRECTORY}/generatebundlefile/generatebundlefile` instead? I don't see bin folder generated on my side.

File: generatebundlefile/hack/generate_bundle.sh [hunk 3]
```diff
@@ -35,6 +35,17 @@ mkdir -p ${BASE_DIRECTORY}/bin
...
+
+# 1.22 Bundle Build
+cd ${BASE_DIRECTORY}/generatebundlefile
+
+${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
+    --input ${BASE_DIRECTORY}/generatebundlefile/data/input_122.yaml \
+    --key alias/${KMS_KEY}
+
+cd ${BASE_DIRECTORY}/generatebundlefile/output
+${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1-22-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
```
Comment (ewollesen) [inline generatebundlefile/hack/generate_bundle.sh:50]: We're exposing passwords to the process table here.

Comment (ewollesen) [reply generatebundlefile/hack/generate_bundle.sh:50]: There are lots of these. I'm making a ticket to deal with this asap after the release. At least these tokens are short-lived and only visible on AWS-controlled machines (and well, the machine of anyone silly enough to clone and run this code).

**Review by ewollesen — APPROVED** [review]
/lgtm

File: generatebundlefile/hack/release.sh (hunk 2 above)
Comment (jonahjon) [reply generatebundlefile/hack/release.sh:49]: The Make build creates a bin folder

File: generatebundlefile/hack/generate_bundle.sh (hunk 1 above)
Comment (jonahjon) [reply generatebundlefile/hack/generate_bundle.sh:40]: Yea Agree, a list of version is probably better.

File: generatebundlefile/hack/generate_bundle.sh (hunk 3 above)
Comment (jonahjon) [reply generatebundlefile/hack/generate_bundle.sh:50]: There is an issue about using the ORAS with docker cred helper, but never figured it out.

**Pull request merged**